### PR TITLE
Feature/op categories

### PR DESCRIPTION
**Pull request by girishranganathan**

### CODE REVIEW

File: istio/istio.go [hunk 1]
```diff
@@ -439,13 +444,16 @@ func (iClient *IstioClient) ApplyOperation(ctx context.Context, arReq *meshes.Ap
 				opName = "removed"
 			}
 			iClient.eventChan <- &meshes.EventsResponse{
-				EventType: meshes.EventType_INFO,
-				Summary:   fmt.Sprintf("Book Info app %s successfully", opName),
-				Details:   fmt.Sprintf("The Istio canonical Book Info app is now %s.", opName),
+				OperationId: arReq.OperationId,
+				EventType:   meshes.EventType_INFO,
+				Summary:     fmt.Sprintf("Book Info app %s successfully", opName),
+				Details:     fmt.Sprintf("The Istio canonical Book Info app is now %s.", opName),
```
Comment (leecalcote) [inline istio/istio.go]: Will we have a function specific to each sample app?

File: istio/supported_ops.go [hunk 2]
```diff
@@ -34,34 +37,42 @@ const (
...
+		opType: meshes.OpCategory_INSTALL,
 	},
 	installmTLSIstioCommand: {
-		name: "Install the latest version of Istio with mTLS",
+		name:   "Latest Istio with mTLS",
+		opType: meshes.OpCategory_INSTALL,
 	},
 	installBookInfoCommand: {
-		name: "Install the canonical Book Info Application",
+		name: "Canonical Book Info Application",
```
Comment (leecalcote) [inline istio/supported_ops.go]: Perhaps, displace "canonical" with "sample".

File: istio/istio.go (hunk 1 above)
Comment (girishranganathan) [reply istio/istio.go]: Not necessarily. Only the ones which need extra care probably have a separate method.

Comment (leecalcote) [reply istio/istio.go]: Good. Ideally, not. Assuming we can generic-ize, so should we keep the message/string generic where possible.

File: istio/supported_ops.go (hunk 2 above)
Comment (girishranganathan) [reply istio/supported_ops.go]: I can drop `canonical`. But we probably don't want `sample` because it already belongs to sample application category.

**Pull request merged**